<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportMegaLinksDialog</class>
 <widget class="QDialog" name="ImportMegaLinksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open links</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ImportMegaLinksDialog 
{
background-color: #FFFFFF; /*colorToken.page-background*/
}

#linkList
{
background-color: #F7F7F7; /*colorToken.surface-1*/
padding-left: 24px;
padding-right: 5px;
border: none;
}

#linkList::item:selected 
{
background: transparent;
}

#linkList::item:selected:active 
{
background: transparent;
}
#linkList::item:selected:!active 
{
background: transparent;
}

#cDownload, #cImport
{
color: #F3F4F4; /*colorToken.text-primary*/
}



</string>
  </property>
  <property name="TOKENIZED" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="QListWidget" name="linkList">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>32</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>33</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>17</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="cDownload">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Download to my computer</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="type" stdset="0">
              <string notr="true">mega</string>
             </property>
             <property name="semibold" stdset="0">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>12</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="eLocalFolder">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="type" stdset="0">
              <string notr="true">mega</string>
             </property>
             <property name="dimension" stdset="0">
              <string notr="true">medium</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bLocalFolder">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Choose</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="type" stdset="0">
              <string notr="true">secondary</string>
             </property>
             <property name="dimension" stdset="0">
              <string notr="true">medium</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="cImport">
             <property name="text">
              <string>Import to my cloud drive</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="type" stdset="0">
              <string notr="true">mega</string>
             </property>
             <property name="style" stdset="0">
              <string notr="true">semibold</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wImport" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>12</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="eMegaFolder">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="type" stdset="0">
              <string notr="true">mega</string>
             </property>
             <property name="dimension" stdset="0">
              <string notr="true">medium</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bMegaFolder">
             <property name="text">
              <string>Choose</string>
             </property>
             <property name="type" stdset="0">
              <string notr="true">secondary</string>
             </property>
             <property name="dimension" stdset="0">
              <string notr="true">medium</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bLayout" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="type" stdset="0">
         <string notr="true">secondary</string>
        </property>
        <property name="dimension" stdset="0">
         <string notr="true">medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bOk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="type" stdset="0">
         <string notr="true">primary</string>
        </property>
        <property name="dimension" stdset="0">
         <string notr="true">medium</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cDownload</tabstop>
  <tabstop>eLocalFolder</tabstop>
  <tabstop>bLocalFolder</tabstop>
  <tabstop>cImport</tabstop>
  <tabstop>eMegaFolder</tabstop>
  <tabstop>bMegaFolder</tabstop>
  <tabstop>bCancel</tabstop>
  <tabstop>bOk</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>ImportMegaLinksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
